src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
Alright, I will add a reference for the ScrollPane in UI or PanelControl.

The MenuControlTest tests the items in menu control, like Add Panel, Save Board, Open Board, Remove Repo. You want to rename it to MenuControlItemTest?